packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -148,6 +176,8 @@ export class Cluster extends Resource implements ICluster {
 
     this.vpc = props.vpc || new ec2.Vpc(this, 'Vpc', { maxAzs: 2 });
 
+    this.capacityProviders = props.capacityProviders;
```

Comment:
Should CapacityProviders also contain the name of the capcity providers in defaultCapacityProviderStrategy?

Reply:
Hm, yeah there might be a way to make that nicer using `Lazy` -- @MrArnoldPalmer can you confirm?